tests/functional/graph_selection/test_schema_test_graph_selection.py

```diff
@@ -12,7 +12,7 @@ def run_schema_and_assert(project, include, exclude, expected_tests):
     run_dbt(["deps"])
     run_dbt(["seed"])
     results = run_dbt(["run", "--exclude", "never_selected"])
-    assert len(results) == 10
+    assert len(results) == 12
```

Comment:
added 2 (non-ephemeral) models to the fixtures [here](https://github.com/dbt-labs/dbt-core/pull/7351/files#diff-a12c5c36034799ba76ea9118119d6cc4966a66714899dea076f76d1ffaf5c31fR217-R221)